subprojects/kotlinReleaseCoroutinesTest/kotlinReleaseCoroutinesTest.gradle

```diff
@@ -0,0 +1,26 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.0-rc-57"
+    }
+}
+
+apply plugin: "org.jetbrains.kotlin.jvm"
+
+description = "Kotlin tests for Mockito."
+
+repositories {
+    mavenCentral()
+    maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
```

Comment:
While I do understand we are breaking soon, I would rather wait for an official release before making any changes. (I do want to incorporate change, just when the package has been released).

Reply:
This also leaves the question whether we should maintain 2 versions of our Kotlin tests. Maybe we should just have one and state that we only support the latest version? WDYT @mockito/core ?

Reply:
@TimvdLippe Just for clarification, this dependency is test-only and won't be visible for library users.
The relevant change is at https://github.com/mockito/mockito/pull/1501/files#diff-67883a6458cc322a1cf66cbaa8b50b1eL14
And it would be very cool if in the moment of release we had a version of mockito that would work both with old and new kotlin compiler versions.

Thanks!

Reply:
Yes that is true. It is mostly a question regarding compatibility. How long do we want to maintain 2 projects to ensure we remain compatible. I think that is a question the core team should answer to resolve this situation.

Reply:
@TimvdLippe This really is a pressing issue - no reply from the core team for 7 days is quite bad. Can you check again? :)

Reply:
I have requested their opinion again. Is there a timeline for release of Kotlin 1.3?

Reply:
@TimvdLippe We expect it in a few weeks. And since people are already struggling after updating to 1.3-rc getting their test failing for unknown reasons it would be cool to merge it sooner than later

Reply:
This looks good to me. 

On a side note, we always take the time we need. This is a free time project for us after all and you cannot expect a reply within five work days. We just got Java 11 support out of the door and if Kotlin decides to break its APIs this is on them and not on us, they must expect such problems when evolving the language that way.